testing/spectest/shared/fulu/fork/upgrade_to_fulu.go

```diff
@@ -38,7 +39,7 @@ func RunUpgradeToFulu(t *testing.T, config string) {
 			}
 			preState, err := state_native.InitializeFromProtoElectra(preStateBase)
 			require.NoError(t, err)
-			postState, err := fulu.UpgradeToFulu(preState)
+			postState, err := fulu.UpgradeToFulu(context.Background(), preState)
```

Comment:
```suggestion
			postState, err := fulu.UpgradeToFulu(t.Context(), preState)
```

You'll have to run goimports too